src/plugins/intel_gpu/include/intel_gpu/primitives/resample.hpp

```diff
@@ -112,6 +112,36 @@ struct resample : public primitive_base<resample> {
           coord_trans_mode(ctm),
           round_mode(nm) {}
 
+    /// @brief resample with dynamic sizes/scales
+    resample(const primitive_id& id,
+             const input_info& input,
+             const input_info& sizes_or_scales_id,
+             const std::vector<int64_t>& axes,
+             const std::vector<size_t>& pads_begin = {},
+             const std::vector<size_t>& pads_end = {},
+             int32_t antialias = 0,
+             float cube_coeff = -0.75f,
+             InterpolateOp::InterpolateMode operation_type = InterpolateOp::InterpolateMode::LINEAR,
+             InterpolateOp::ShapeCalcMode shape_calc_mode = InterpolateOp::ShapeCalcMode::SIZES,
+             InterpolateOp::CoordinateTransformMode ctm = InterpolateOp::CoordinateTransformMode::HALF_PIXEL,
+             InterpolateOp::NearestMode nm = InterpolateOp::NearestMode::ROUND_PREFER_FLOOR)
```

Comment:
Try to add paddings to this constructor as it is done in other constructors
Maybe this will solve the problem with prepare_buffer_fusing